.github/workflows/gradle.yml

```diff
@@ -33,4 +33,4 @@ jobs:
         java-version: ${{ matrix.java }}
         cache: 'gradle'
     - name: Build with Gradle
-      run: ./gradlew clean build -x unitTest -x checkstyleTest
```

Comment:
how/where checkstyle is executed now?

Reply:
Gradle flag `-x` means exclusion of task from the build, in other words `-x checkstyleTest` disabled Checkstyle checks against test sources.

Now this exclusion is moved to Gradle build config.